src/current/v24.1/disaster-recovery-overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Disaster Recovery Overview
+summary: Learn how to make your cluster resilient with high availability and disaster recovery features.
+toc: true
+---
+
+Resilient deployments aim for continuity in database operation to protect from data loss and down time. To maintain resiliency, it is necessary to build deployments with _high availability_ and _disaster recovery_ coverage.
```

Comment:
database operation continuity?

Reply:
I haven't changed this. Having "database operation" modify "continuity" feels a little harder to read. I have left as-is for now — hopefully my docs review partner may have an idea here.

Reply:
No worries, sounds good!